### PR TITLE
Added fields for task levels

### LINKED ISSUE #189: Add two new fields in the `Create Task` form
Add the following data fields required for the Leveling of Tasks
- Add a category field
- Add a level field

For reference : https://github.com/Real-Dev-Squad/todo-action-items/issues/72

### PR DESCRIPTION
**Pull request by rohan09-raj**
Closes #189

### CODE REVIEW

File: src/task/index.html [hunk 1]
```diff
@@ -106,6 +106,38 @@
...
+              Category:<span class="required">*</span>
+            </label>
+            <select
+              required
+              class="input notEditing"
+              id="category"
+              name="category"
+            >
+              <option selected value="system_design">System Design</option>
+              <option selected value="frontend">Frontend</option>
```
Comment (Mir-SA) [inline src/task/index.html:123]: Why are both fields selected?
Also this dropdown currently have only 4 fields, how will we add `other` skill? Suppose a custom skill like `React`

File: src/task/style.css [hunk 2]
```diff
@@ -104,6 +104,8 @@ body {
   font-size: medium;
 }
 
+#category,
+#level,
 #status {
```
Comment (shubham-y) [inline src/task/style.css:109]: Isn't it better to have a class for it?

Comment (rohan09-raj) [reply src/task/style.css:109]: The class is already there and it is applied to many more elements.
That class is used in JS for a separate purpose
The code needs refactoring and I guess @Pavangbhat is already taking care of it separately

**Review by shubham-y — APPROVED** [review]

**Review by Pavangbhat — DISMISSED** [review]

File: src/task/index.html [hunk 3]
```diff
@@ -106,6 +106,37 @@
...
+            </label>
+            <select
+              required
+              class="input notEditing"
+              id="category"
+              name="category"
+            >
+              <option selected value="system_design">System Design</option>
+              <option value="frontend">Frontend</option>
+              <option value="backend">Backend</option>
```
Comment (vinayak-trivedi) [inline src/task/index.html:124]: can we make all the letters uppercase in the category to maintain consistency?

Comment (rohan09-raj) [reply src/task/index.html:124]: The labels are consistent.
We have the same in status also
image

Comment (rohan09-raj) [reply src/task/index.html:124]: Are you concerned about the values ??

Comment (vinayak-trivedi) [reply src/task/index.html:124]: yes, all the values of status are also stored in uppercase in our DB

**Review by vinayak-trivedi — APPROVED** [review]
Thanks for doing the changes, LGTM 👍

**Pull request merged**